FREEPBX-20119

vsc55 opened the pull request:
add support for configuration port mysql server custom

qwell commented:
This may work well for your scenario, but to be included, this would need to be part of a much bigger improvement to add AMPDBPORT in all of the appropriate places.  Any place that AMPDBHOST is used  (15-20 in framework) will likely need to be changed to also use AMPDBPASS.  This would include setting a proper default of 3306 in the installer (installcommand.class.php).

It's a scary change that would need a great deal of regression testing.

vsc55 commented:
Hi Qwell,
I have updated the rest of the files that refer to the connection with the database (it seems to me that they all do).
I have been tested and do not see errors in "asterisk" or in "fwconsole debug".
**Pending test installation.**

Note:
I have modified some files that are inside "/upgrades", I do not know if they have to be updated or not.
